style.css

```diff
@@ -106,3 +106,277 @@ body {
   padding: 8px;
   border-radius: 8px;
 }
+
+.works-section {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  width: 100%;
+  gap: 24px;
+  justify-content: center;
+}
```

Comment:
Great job @Enning94 🤝

- [ ] You may please align the contents of the works section with the display grid instead of flex. Please check out the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) for more clarity.